cubedash/static/base.css

```diff
@@ -1158,6 +1158,11 @@ details > summary {
   max-height: none !important;
 }
 
+/* Unset image overlay width to preserve correct image aspect ratio */
+.leaflet-container img.leaflet-image-layer {
```

Comment:
Please put in the change to `.scss` file

Reply:
What `.scss` file are you referring to?

Reply:
this one https://github.com/opendatacube/datacube-explorer/blob/develop/cubedash/static/base.sass

Reply:
When I run `make style` it doesn't do anything

Reply:
Done